src/services/config.js

```diff
@@ -103,8 +108,13 @@ class ConfigData {
       if (this.activeProfile) {
         profile = this.getProfileFromConfigFileById(this.activeProfile);
       }
+
+      // TODO: Ensure order of profiles : DI-1479
       if (!profile) {
-        profile = this.projects[0];
+        profile = this.projects[0] || Object.values(this.profiles)[0];
+        if (profile && !profile.hasOwnProperty('id')) {
```

Comment:
`profile` will always be there because of [this](https://github.com/twilio/twilio-cli-core/pull/124/files#diff-69b5d67d841057efec105a48299840e180ffc94f0da052f105dd16f5465d6a9cR107).